components/brave_new_tab_ui/actions/grid_sites_actions.ts

```diff
@@ -6,36 +6,28 @@
 // Types
 import { types } from '../constants/grid_sites_types'
 import { action } from 'typesafe-actions'
-import { InitialData } from '../api/initialData'
 
-export const setFirstRenderGridSitesData = (initialData: InitialData) => {
-  return action(types.GRID_SITES_SET_FIRST_RENDER_DATA, initialData)
-}
-
-export const gridSitesDataUpdated = (gridSites: NewTab.Site[]) => {
+export const tilesUpdated = (gridSites: NewTab.Site[]) => {
   return action(types.GRID_SITES_DATA_UPDATED, { gridSites })
```

Comment:
IMO we should rename these consts to match the renamed exported function names, so that searching for the reducer handler is easier (camelCase -> UPPER_CASE)

Reply:
Can we do that in a separate PR? (ex: follow up) or I can do the ones I changed now if we'd like?

Reply:
I'm leaning towards thinking we should do the simple find and replace now because AFAIK every single reducer action and type name matches exactly (apart from the casing). It might lead to some bugs because a dev uses search incorrectly.

Reply:
Checking this out now

Reply:
Should be fixed now 😄